lib/common.sh

```diff
@@ -733,8 +682,8 @@ common::check_packages () {
             "Run the following and try again:"
     logecho
     for prereq in ${missing[@]}; do
-      if [[ $prereq == "sendgmr" ]] && [[ $distro == "Ubuntu" ]]; then
-        logecho "sudo goobuntu-add-repo $prereq && sudo apt-get update"
+      if [[ ${PREREQUISITE_INSTRUCTIONS[$prereq]} ]]; then
+        logecho "go/$prereq"
```

Comment:
I think you wanted to `logecho $PREREQUISITE_INSTRUCTIONS[$prereq]` here instead?